editor/animation/animation_track_editor.cpp

```diff
@@ -5391,6 +5401,14 @@ void AnimationTrackEditor::_notification(int p_what) {
 		case NOTIFICATION_VISIBILITY_CHANGED: {
 			update_keying();
 		} break;
+
+		case NOTIFICATION_TRANSLATION_CHANGED: {
+			bezier_key_mode->set_item_text(bezier_key_mode->get_item_index(Animation::HANDLE_MODE_FREE), TTR("Free", "Bezier Handle Mode"));
+			bezier_key_mode->set_item_text(bezier_key_mode->get_item_index(Animation::HANDLE_MODE_LINEAR), TTR("Linear", "Bezier Handle Mode"));
+			bezier_key_mode->set_item_text(bezier_key_mode->get_item_index(Animation::HANDLE_MODE_BALANCED), TTR("Balanced", "Bezier Handle Mode"));
+			bezier_key_mode->set_item_text(bezier_key_mode->get_item_index(Animation::HANDLE_MODE_MIRRORED), TTR("Mirrored", "Bezier Handle Mode"));
+			bezier_key_mode->set_tooltip_text(TTR("Bezier Default Mode\nSet the default handle mode of new bezier keys."));
```

Comment:
This probably needs an accessibility text too, doesn't it? Like, the first line of the tooltip, like @bruvzg seems to have done in other accessibility PRs.

You could be smart with this, and maybe use `String.get_slicec('\n', 0)` to obtain the accessibility text. Hmm...